go.mod

```diff
@@ -18,20 +18,20 @@ require (
 	github.com/golang/protobuf v1.3.2
 	github.com/heptio/authenticator v0.0.0-20180409043135-d282f87a1972
 	github.com/pkg/errors v0.8.1
-	github.com/rancher/norman v0.0.0-20200302172514-1ee4db8c9219
-	github.com/rancher/rke v1.1.0-rc11
-	github.com/rancher/types v0.0.0-20200304001827-068a357fa053
+	github.com/rancher/norman v0.0.0-20200326201949-eb806263e8ad
+	github.com/rancher/rke v1.1.0-rc9.0.20200327175519-ecc629f2c3d5
```

Comment:
Need to update this after merging https://github.com/rancher/rke/pull/2060 with the commit from the release/v1.1 branch